google/cloud/spanner/commit_options.h

```diff
@@ -70,6 +70,16 @@ class CommitOptions {
     return request_priority_;
   }
 
+  CommitOptions& set_max_batching_delay_ms(
```

Comment:
@devbww do we want to keep adding methods to this class?  Can we support this new feature *only* via `google::cloud::Options{}`?

Reply:
No to `spanner::CommitOptions`.  Yes, to `google::cloud::Options`.